vello/src/debug/renderer.rs

```diff
@@ -211,8 +213,10 @@ impl DebugRenderer {
         }
     }
 
-    #[allow(clippy::too_many_arguments)]
-    // #[expect(clippy::too_many_arguments, reason="This function is internal, so the argument count doesn't cause issues for consumers.")]
+    #[expect(
+        clippy::too_many_arguments,
+        reason = "This function is internal, so the argument count doesn't cause issues for consumers."
+    )]
```

Comment:
Not sure what value `clippy::too_many_arguments` brings to the table.

Reply:
Yeah, but the number of times I've gotten that lint and thought "Oh, I'll restructure the code" vs the number of times that we suppress it ...

Reply:
It's true that an actually unreasonable number of arguments would be caught by human review.

A compromise could be to increase our [too-many-arguments-threshold](https://doc.rust-lang.org/clippy/lint_configuration.html#too-many-arguments-threshold) from 7 to something higher.

Reply:
I think it's something good to reason about for externally facing APIs, but for this kind of internal stuff ignoring it is fine. I don't think there's a way to configure the two separately.

What solution do I need to do here to get this PR landable?

Reply:
I think this can remain as-is for this PR, and let's come up with a Linebender-wide solution in [Zulip](https://xi.zulipchat.com/#narrow/channel/419691-linebender/topic/Standard.20Lint.20set).